nix/modules/holochain-crate2nix.nix

```diff
@@ -6,6 +6,8 @@
       };
       customBuildRustCrateForPkgs = pkgs: pkgs.buildRustCrate.override {
         defaultCrateOverrides = pkgs.defaultCrateOverrides // {
+          tx5-go-pion-sys = _: { nativeBuildInputs = with pkgs; [ go ]; };
+          tx5-go-pion-turn = _: { nativeBuildInputs = with pkgs; [ go ]; };
```

Comment:
So opaque! I never would have come up with this. Thanks for fixing it!

Reply:
Opaque squared

Reply:
it's actually quite nicely documented :blush: https://github.com/kolloch/crate2nix#patching-crate-derivations-with-crateoverrides

Reply:
coupled with what i explained in a comment above:

> toolchains go only into the `nativeBuildInputs`. see https://nixos.org/manual/nixpkgs/stable/#var-stdenv-nativeBuildInputs